controller_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: bash ./controller_coverage.sh
+# Then run: firefox controller_coverage/index.html
+
+set -e
+set -x
+
+# delete coverage files
+#find .pio -name "*.gcno" -delete
+
+# run tests
+pio test -e native
+
+ENVIRONMENT=native
+OUTPUT_DIR=controller_coverage
+rm -rf $OUTPUT_DIR
```

Comment:
This rm -rf is very powerful, so to respect that power, I think we should quote the variable `$OUTPUT_DIR`.  Might as well quote other variables too.